### DISCUSSION TITLE
Stuck at "Processing information…"

### DISCUSSION THREAD

**Discussion by aersoares81** [community/community#86597] [GitHub Education] [answered]
### Hi everyone,

When I submit my proof of academic status the green button greys out and stay at "Processing information…" forever, nothing really happens. I've tried Chromium and Safari browsers.

**Comment by dariohsb** [comment]
Disable the adblock if you have one, delete the cache and cookies from the history and try again, if not, try doing it from incognito.

**Reply by OtmaneDaoudi** [reply]
Not working

**Reply by aersoares81** [reply]
I tried that, didn't help. I used another browser even…

**Reply by Loose-Gu** [reply]
the same, the post request of the proof file returns 413 error code

**Reply by tzuntar** [reply]
Apparently the file's too large hence the 413. It works for a <150KB JPEG. Although this makes any kind of document scan illegible. *Definitely a bug as it even says it should accept files of up to 10MB in size.*

**Reply by mishmanners** [reply, marked as answer]
The same is resolved here: https://github.com/orgs/community/discussions/88401#discussioncomment-8142514 and here: https://github.com/orgs/community/discussions/88446#discussioncomment-8148738

Please see these. The TL;DR:
- Refresh the page or clear your browser's cache and cookies
- Check for browser extensions that could be interfering with the JavaScript code and disable them
- Try a different browser
- Ensure your file is less than 150kB

**Comment by thistlillo** [comment]
I have the same issue (and tried the solution proposed here with no success). I applied using MacOS and two different browsers: Chrome and Safari.

**Comment by EssKei** [comment]
I Used a online Compressor to make the Pictures smaller, worked

**Reply by OtmaneDaoudi** [reply]
This one worked for me! thank you!

**Reply by hyphy-void** [reply]
thank you

**Comment by thistlillo** [comment]
I confirm. After trying a couple of times with a smaller image, I was able to upload one.

**Comment by JasonMendoza2008** [comment]
I confirm, using a smaller image was the way to go

**Comment by afeuerstein** [comment]
Seems to be the same problem as here: https://github.com/orgs/community/discussions/88401#discussioncomment-8148320
(#88401)
